website/project/forms.py

```diff
@@ -16,4 +17,5 @@ class NewNodeForm(Form):
         validators.Length(max=200, message=u'Title must contain fewer than 200 characters.')
     ], widget=BootstrapTextInput())
     description = TextAreaField('Description', widget=BootstrapTextArea())
-    category = TextAreaField('Category', widget=BootstrapTextArea())
+    category = TextField('Category', widget=BootstrapTextInput())
+    inheritContributors = TextField('Inherit', widget=BootstrapTextInput())
```

Comment:
We should understand and remove the Python bits of `NewNodeForm` that are extraneous.